.rubocop.yml

```diff
@@ -6,18 +6,18 @@ require:
 
 AllCops:
   Include:
-    - '**/Gemfile'
-    - '**/config.ru'
-    - '**/Rakefile'
-    - '**/*.rake'
-    - '**/*.rb'
+    - "**/Gemfile"
```

Comment:
Is this a result of a setting in VSCode causing this to happen?

Reply:
I believe so, will try disabling my plugin for formatting